src/main/java/core/basesyntax/files/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
You should provide a more informative message in the RuntimeException. For example, you can include the filePath in the message to make it clear which file could not be read.